app/assets/stylesheets/layout/_footer.scss

```diff
@@ -0,0 +1,12 @@
+footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  justify-content: center;
+  margin-top: 2rem;
+  padding: 1rem;
+
+  p {
+    color: $white;
+  }
+}
```

Comment:
add a blank line at the end of this file too.

Reply:
done  :+1: